tutorials/AMB2018-02-B/system/ExaCA

```diff
@@ -10,32 +10,6 @@ ExaCA
 {
     type ExaCA;
     libs ("libExaCAFunctionObject.so");
-
-    scalarData      5;
-    labelData      1;
-
-/*
-    writeControl    timeStep;
-    writeInterval   1;
-
-    setFormat csv;
-
-    interpolationScheme pointMVC;
-
-    sets
-    (
-        data
-        {
-            type        boxUniform;
-            axis        xyz;
-            box         (0 -0.0001 -0.0002) (0.002 0.0001 0);
```

Comment:
where is the bounding box information kept in the new ExaCA FunctionObject?

Reply:
or is it no longer needed? is matt doing that part of the conversion in ExaCA itself?

Reply:
The ExaCA FunctionObject is just returning the temperature at vertices at the moment. I will need to work on the logic to actually interpolate and write the data to exaca points. This comment was removed in [fbba891](https://github.com/kincaidkc/AdditiveFOAM/pull/25/commits/fbba891ccd5b5200279a1453f7588b0c9fb7c7eb)